recipe/meta.yaml

```
@@ -12,13 +12,14 @@ source:
 build:
   noarch: python
   number: 0
-  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  script: "{{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv"
 
 requirements:
   host:
     - pip
     - python >=3.5
```

```suggestion
    - python
```

If not noarch, remove python pins.